addons/mail/views/res_partner_views.xml

```diff
@@ -26,6 +26,24 @@
             </field>
         </record>
 
+        <record id="res_partner_view_kanban_inherit_mail" model="ir.ui.view">
+            <field name="name">res.partner.view.kanban.inherit.mail</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="base.res_partner_kanban_view"/>
+            <field name="arch" type="xml">
+                <field name="type" position="after">
+                    <field name="activity_ids"/>
+                    <field name="activity_state"/>
+                    <progressbar field="activity_state" colors='{"planned": "success", "today": "warning", "overdue": "danger"}'/>
+                </field>
```

Comment:
I fail to see what this does. Could you explain?

Reply:
The field 'activity_state' will show the status like Overdue,Today,Planned based on activities. Yes, you are right there is no need of progressbar in this kanban so i removed it. Thanks.!

Reply:
Maybe it's doesn't, but wouldn't be nicer to just add this field without relying on another one? Like this:
```xml
<kanban position="inside">
   <field name="activity_state"/>
</kanban>
```
From my understanding, this would be a more robust solution. What do you think?

Reply:
Yes, you are right. This solution is better than previous one. I will change it.